unixfs/Cargo.toml

```diff
@@ -9,7 +9,9 @@ repository = "https://github.com/rs-ipfs/rust-ipfs"
 version = "0.0.1"
 
 [features]
-default = ["filetime"]
+default = []
+std = []
+with-filetime = ["filetime"]
```

Comment:
is creating a `with-` prefixed feature for a dependency a better practice than just using the deps as features? I can't remember seeing this style.

Reply:
This is a generic pattern that avoids conflicts with the dependency name and is useful for features that have more than one dependency.

For example, `with-serde = ['serde', 'serde_derive']` (works) instead of `serde = ['serde', 'serde_derive']` (error).

Reply:
Thanks for clarifying that up, sounds like the way to go.